### DISCUSSION TITLE
Distant Horizons Support?

### DISCUSSION THREAD

**Discussion by samin50** [gri573/rethinking-voxels#317] [General]
Hi, are there any plans to add Distant Horizons support? Its gaining some traction now and Complimentary Shaders now includes support for it, I would love to see this shaderpack also support it so I can use it as my everyday shader.

**Comment by heide-oficial** [comment]
Yes! Please! That would be amazing!

**Comment by Radstark** [comment]
Would be really great to have.

**Comment by Trimatix** [comment]
Going to comment both here and in the voxy thread - it seems that Complementary Reimagined implemented Voxy/Distant Horizons support in v5.8, so if this fork is updated, support should come for ""free"" (i imagine updating the fork is non trivial!)
